src/services

```diff
@@ -1147,3 +1147,20 @@ p({
       }
     """
 })
+
+p({
+  "class": "foam.nanos.boot.NSpec",
+  "name": "timezoneDAO",
+  "description": "A DAO which contains information about timezones in countries.",
+  "serve": true,
+  "serviceScript": """
+    return new foam.dao.EasyDAO.Builder(x)
+      .setPm(true)
+      .setAuthorizer(new foam.nanos.auth.GlobalReadAuthorizer("timezoneDAO"))
+      .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
+      .setJournalName(\"timezones\")
```

Comment:
```suggestion
      .setJournalName("timezones")
```

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/3804/commits/da6e69ac634373d89d066043c1bd4f23f17dc4d5.